guide/ops/troubleshooting/web-console-issues.md

```diff
@@ -21,3 +21,19 @@ For more information, see
 [http://stackoverflow.com/questions/23679968/chrome-hangs-after-certain-amount-of-data-transfered-waiting-for-available-soc](http://stackoverflow.com/questions/23679968/chrome-hangs-after-certain-amount-of-data-transfered-waiting-for-available-soc).
 
 [chrome://net-internals/#sockets](chrome://net-internals/#sockets) is also a useful diagnostic tool.
+
+## Windows Defender may report a false-postive virus definiton match when downloading br.exe
+
+On some versions of the Windows Defender malware definitions, the `br.exe` download was erroneously
+flagged as containing a virus. The executable has been scanned against multiple antivirus protection
+providers and no infection has been found. As of 15 Sept. 2021, Microsoft has confirmed that the detection
+has been removed
+
+Please follow the steps below to clear cached detection and obtain the latest malware definitions.
+
+1. Open command prompt as Administrator and change directory to `C:\Program Files\Windows Defender`
+2. Run `MpCmdRun.exe -removedefinitions -dynamicsignatures`
+3. Run `MpCmdRun.exe -SignatureUpdate`
+
+Alternatively, the latest definition is available for download here: https://www.microsoft.com/en-us/wdsi/definitinos
```

Comment:
URL incorrect, should either end in `definitions` or use https://www.microsoft.com/en-us/wdsi/defenderupdates and surround with square brackets